src/SignalR/common/Http.Connections/src/HttpConnectionDispatcherOptions.cs

```diff
@@ -68,6 +71,26 @@ public HttpConnectionDispatcherOptions()
         /// </summary>
         public int MinimumProtocolVersion { get; set; } = 0;
 
+
+        /// <summary>
+        /// Gets or sets the amount of time the transport will wait for a send to complete. If a single send exceeds this timeout
+        /// the connection is closed.
+        /// </summary>
+        /// <remarks>
+        /// The default timeout is 10 seconds.
+        /// </remarks>
+        public TimeSpan TransportSendTimeout
```

Comment:
For API review, we can configure `Timeout.InfiniteTimespan` if we want to disable this timeout. This should be documented.